src/scripts/i18n/inline_and_diff.py

```
@@ -0,0 +1,201 @@
+        kebab = kebab[len('feature-'):]
+    if kebab.startswith('ui-'):
+        kebab = kebab[len('ui-'):]
+    return kebab_to_camel(kebab)
+
+interpolation_re = re.compile(r"\{\{\s*(['\"])([\w.]+)\1\s*\|\s*transloco\s*\}\}")
+html_bind_re = re.compile(r"(['\"])([\w.]+)\1\s*\|\s*transloco")
+ts_call_re = re.compile(r"(?:\w+\.)?translocoService\.translate\(\s*(['\"])([\w.]+)\1\s*(?:,\s*\{([^}]*)\})?\s*\)")
+attr_binding_redundant_quotes = re.compile(r'(\[[\w-]+\])="\'([^\']*)\'"')
+str_double_quote_re = re.compile(r':\s*"([^"]*?)"')
+
+def replace_in_file(path: str, content: str) -> str:
```

Ruff (0.11.9)

`replace_in_file` is too complex (17 > 10)

(C901)